tsl/test/shared/expected/compression_dml.out

```diff
@@ -605,46 +605,42 @@ ROLLBACK;
 BEGIN; :ANALYZE DELETE FROM direct_delete WHERE value = '1.0'; ROLLBACK;
 QUERY PLAN
  Custom Scan (ModifyHypertable) (actual rows=0 loops=1)
-   Batches decompressed: 8
-   Tuples decompressed: 8
+   Batches deleted: 8
    ->  Delete on direct_delete (actual rows=0 loops=1)
          Delete on _hyper_X_X_chunk direct_delete_1
-         ->  Seq Scan on _hyper_X_X_chunk direct_delete_1 (actual rows=8 loops=1)
+         ->  Seq Scan on _hyper_X_X_chunk direct_delete_1 (actual rows=0 loops=1)
                Filter: (value = '1'::double precision)
-(7 rows)
+(6 rows)
 
 BEGIN; :ANALYZE DELETE FROM direct_delete WHERE device = 'd1' AND value = '1.0'; ROLLBACK;
 QUERY PLAN
  Custom Scan (ModifyHypertable) (actual rows=0 loops=1)
-   Batches decompressed: 4
-   Tuples decompressed: 4
+   Batches deleted: 4
    ->  Delete on direct_delete (actual rows=0 loops=1)
          Delete on _hyper_X_X_chunk direct_delete_1
-         ->  Seq Scan on _hyper_X_X_chunk direct_delete_1 (actual rows=4 loops=1)
+         ->  Seq Scan on _hyper_X_X_chunk direct_delete_1 (actual rows=0 loops=1)
                Filter: ((device = 'd1'::text) AND (value = '1'::double precision))
-(7 rows)
+(6 rows)
 
 BEGIN; :ANALYZE DELETE FROM direct_delete WHERE reading = 'r1' AND value = '1.0'; ROLLBACK;
 QUERY PLAN
  Custom Scan (ModifyHypertable) (actual rows=0 loops=1)
-   Batches decompressed: 2
-   Tuples decompressed: 2
+   Batches deleted: 2
    ->  Delete on direct_delete (actual rows=0 loops=1)
          Delete on _hyper_X_X_chunk direct_delete_1
-         ->  Seq Scan on _hyper_X_X_chunk direct_delete_1 (actual rows=2 loops=1)
+         ->  Seq Scan on _hyper_X_X_chunk direct_delete_1 (actual rows=0 loops=1)
```

Comment:
It should say "rows=2", right? Maybe we're missing the setup for EXPLAIN context somewhere. There's an example in vectorized aggregation in `compressed_batch_get_next_slot`. It works in conjunction with `compresessed_batch_set_compressed_tuple`, you might need to add more since you're reimplementing it.

Reply:
The problem here is that we are deleting these two rows directly without recompression. I know this looks strange but this only reports deletes of uncompressed tuples.